src/werkzeug/serving.py

```diff
@@ -561,6 +564,7 @@ def make_ssl_devcert(
 
     if host is not None:
         cn = f"*.{host}/CN={host}"
+        cn = host
```

Comment:
You added an extra `cn = ` line rather than replacing the existing line. Setting it to host rather than the previous value is quite different, please explain this change.

Reply:
you're right i should delete the old. 

The format of [cn](https://knowledge.digicert.com/solution/what-is-a-common-name#:~:text=SSL%20Certificates,or%20%22digicert.com%22.) in the cert i think is invalid, it should be a single fully qualified name, when presented to the client as is, the client will error with:

ERROR: no certificate subject alternative name matches
	requested host name ‘ourapp.localhost’.

So i squish it and make it just the localhost, and add localhost, and *.localhost in the SANS extension.